content/release/ReleaseNotes372.md

```diff
@@ -0,0 +1,113 @@
+<!--
+title: "Contrast 3.7.2 - March 2020"
+description: "Contrast 3.7.2 March 2020"
+tags: "3.7.2 March Monthly Release News"
+-->
+
+## On-premises release date:
+March 31, 2020
```

Comment:
Does this date need to be changed - did we really release on 3/31?

Reply:
Thank you Dan.